src/lib/crypto/snarky_tests/snarky_tests.ml

```diff
@@ -434,6 +434,44 @@ let api_tests =
   ; ("compile monadic API", `Quick, MonadicAPI.get_hash_of_circuit)
   ]
 
+module Protocol_circuits = struct
```

Comment:
needs a description of what's happening here `(**`

Reply:
done.